node/sc/multi_bridge_test.go

```diff
@@ -76,3 +76,30 @@ func TestRegisterDeregisterOperator(t *testing.T) {
 	assert.NoError(t, err)
 	assert.Equal(t, isOperator, false)
 }
+
+// TestStartStop checks the following:
+// - the bridge contract method Start.
+// - the bridge contract method Stop.
+func TestStartStop(t *testing.T) {
+	info := prepareMultiBridgeTest(t)
+
+	opts := &bind.TransactOpts{From: info.acc.From, Signer: info.acc.Signer, GasLimit: gasLimit}
+	tx, err := info.b.Start(opts, true)
+	assert.NoError(t, err)
```

Comment:
is this okay not to finish a test if it is an err??

Reply:
It's okay.